### PR TITLE
Create LINKEDIN_BUTTON_BY_Rakesh-116.html

### PR DESCRIPTION
**Pull request by Rakesh-116**

### CODE REVIEW

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
Nice Job. 'You nailed it.'
**Last change...**

Remove the html, head, body, title, doctype tags... Go straight to the solution. Modules are not full web pages but are meant to be imported into standard visual web pages.

e.g

```
<div>
     <!-- Your Solution -->
</div>
```
If you make this changes across all your Pull Request, I would be accepting your PR today...

**Comment by Rakesh-116** [comment]
should i create new file again or i can edit in that file only?

**Comment by Rakesh-116** [comment]
[image: image.png]
it is getting in this red color , if i edit, does that mean wrong, should
close this one and create another file

**Comment by Rakesh-116** [comment]
if it is okay to edit in the same, i have done that, I removed html, head,
body, title.., please do check it

File: modules/buttons/LINKEDIN_BUTTON_BY_Rakesh-116.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        border-width: 0;
+        border-radius:2px;
+        color:#ffffff;
+        font-size:15px;
+        font-weight: 100;
+      }
+    </style>
+    <div>
+        <button class="Linkedin-button">
+            <span style="font-weight: 600;font-size:20px; padding:5px;">in</span>Linkedin
```
Comment (NtemKenyor) [inline modules/buttons/LINKEDIN_BUTTON_BY_Rakesh-116.html:16]: I like this approach. Funny and brief. No svgs. no images.... LOL

**Review by NtemKenyor — APPROVED** [review]

File: modules/buttons/LINKEDIN_BUTTON_BY_Rakesh-116.html (hunk 1 above)
Comment (Rakesh-116) [reply modules/buttons/LINKEDIN_BUTTON_BY_Rakesh-116.html:16]: Ha, as u said less memory needed

**Pull request merged**